packages/apple-pay-integration/src/mocks/apple-pay-button.mock.ts

```diff
@@ -1,7 +1,32 @@
-import { CheckoutButtonInitializeOptions } from '@bigcommerce/checkout-sdk/payment-integration-api';
+import {
+    BuyNowCartRequestBody,
+    CartSource,
+    CheckoutButtonInitializeOptions,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
 
 import { WithApplePayButtonInitializeOptions } from '../apple-pay-button-initialize-options';
 import ApplePayButtonMethodType from '../apple-pay-button-method-type';
+import { getCart } from '@bigcommerce/checkout-sdk/payment-integrations-test-utils';
```

Comment:
This import should be placed under @bigcommerce/checkout-sdk/payment-integration-api import.
Don't forget to add a space between import groups.

Reply:
moved